packages/showcases/core/csf/guidelines/iconography.csf.js

```diff
@@ -14,7 +14,7 @@ export const argTypes = {
     defaultValue: 24,
     control: {
       type: 'select',
-      options: [16, 24, 32, 64],
+      options: [16, 20, 24, 32, 64],
```

Comment:
It is allowed in design?

Reply:
Added it to match with the React component but it seems that only the 16, 24, and 32 pixels are available. It means we should remove the 20 and 64 size ? We should also make this update in the react component ?